src/common/tree_node_serializer.py

```diff
@@ -154,6 +154,8 @@ def tree_node_from_dict(
     for child_attribute in node.blueprint.get_none_primitive_types():
         if child_attribute.name == "_meta_":
             continue
+        if child_attribute.name == "default" and child_attribute.attribute_type == "any":
```

Comment:
Why do we not want to add attributes name default to the node?

Reply:
As far as I can see, this for loop (starting on line 154) is responsible for adding child nodes to the node.

Since the "default" attribute is not part of the PRIMITIVES enum 
(`PRIMITIVES = {"string", "number", "integer", "boolean"}`), the for loop will try to add "default" as a child node.

So in the same way that _meta_ attribute is ignored (line 155-156), we have to ignore the "default" attribute because we don't need to create a child node for it :)